plugins/woocommerce/client/admin/client/dashboard/leaderboards/index.js

```diff
@@ -180,7 +180,7 @@ Leaderboards.propTypes = {
 
 export default compose(
 	withSelect( ( select ) => {
-		const { getItems, getItemsError } = select( ITEMS_STORE_NAME );
+		const { getItems, getItemsError } = select( itemsStore );
```

Comment:
These two selectors are passed to `Leaderboards` component but don't seem to be used. Maybe consider removing them.